pandas/io/xml.py

```diff
@@ -890,6 +1068,20 @@ def read_xml(
         transformation and not the original XML document. Only XSLT 1.0
         scripts and not later versions is currently supported.
 
+    iterparse : dict, optional
```

Comment:
is this a common verb? maybe stream is more descriptive here

Reply:
Good synonym but `iterparse` is well-recognized in the Python ElementTree API.